website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -461,7 +488,7 @@ mutation {
 }
 ```
 
-**Query a categorical dimension on its own**
+#### Query a categorical dimension on its own
```

Comment:
I think this example is unnecessary, too. Users can already see how to query with a single dimension and a categorical dimension in different examples above.

Reply:
i can totally see your point! i do think it might be helpful to have a specific example about this, rather than have a user try to figure it out by seeing the earlier example that queries two metrics. leaving it here if that's ok!